### PR TITLE
update scripting usersguide

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

File: doc/UsersGuide/source/omjulia.rst [hunk 1]
```diff
@@ -209,12 +214,14 @@ Similary we can also use setInputs() to set a value for the inputs during variou
 
 >>> setInputs(mod,"cAi=1")
 
-And then finally we can simulate the model using, The simulate() API can be used in two methods
+And then finally we can simulate the model using, The simulate() API can be used in following methods
```
Comment (adeas31) [inline doc/UsersGuide/source/omjulia.rst]: Rewrite this line to,

```
The model can be simulated using the `simulate` API in the following ways,
```

File: doc/UsersGuide/source/omjulia.rst [hunk 2]
```diff
@@ -78,6 +78,11 @@ The object constructor requires a minimum of 2 input arguments which are strings
 
 >>> ModelicaSystem(mod,"BouncingBall.mo","BouncingBall",["Modelica", "SystemDynamics", "dcmotor.mo"])
 
+-  The fourth input argument (optional), is a keyword argument which is used to set the commandLineOptions e.g.,
```
Comment (adeas31) [inline doc/UsersGuide/source/omjulia.rst]: Change `commandLineOptions` to `command line options`

File: doc/UsersGuide/source/omjulia.rst [hunk 3]
```diff
@@ -209,12 +214,14 @@ Similary we can also use setInputs() to set a value for the inputs during variou
 
 >>> setInputs(mod,"cAi=1")
 
-And then finally we can simulate the model using, The simulate() API can be used in two methods
+And then finally we can simulate the model using, The simulate() API can be used in following methods
   1) without any arguments
-  2) resultfile names provided by user (only filename is allowed and not the location)
+  2) resultfile (keyword argument) - (only filename is allowed and not the location)
```
Comment (adeas31) [inline doc/UsersGuide/source/omjulia.rst]: Use the sphinx numbering instead of manual numbering.

Comment (arun3688) [reply doc/UsersGuide/source/omjulia.rst]: fixed the numbering

File: doc/UsersGuide/source/omjulia.rst (hunk 2 above)
Comment (arun3688) [reply doc/UsersGuide/source/omjulia.rst]: updated

File: doc/UsersGuide/source/omjulia.rst (hunk 3 above)
Comment (adeas31) [reply doc/UsersGuide/source/omjulia.rst]: Squash the commits to one.

**Pull request merged**